docs/conf.py

```diff
@@ -44,7 +44,7 @@
 
 # General information about the project.
 project = 'Zulip'
-copyright = '2015–2020, The Zulip Team'
+copyright = '2015–2021, The Zulip Team'
```

Comment:
(Should this be more specific like the other notices?)

Reply:
I think probably it makes sense to make the docs/ line identical to the rest, including the start year (I think some small portion of the code style / commit guidelines was extracted from the original 2012-2014 Zulip Inc. wiki, so that'd be the more correct date).

Also, I believe the above 2011s are wrong and should be 2012s, since that's when the original Zulip, Inc. was incorporated.

The 2011 error seems to have been introduced in 11a51cf9438b2679ab2160e66ff2e8265d9a029e aka during the week Zulip was prepared to be made OSS; my guess is @lfaraone accidentally replaced 2012 with 2011 in that commit.

Reply:
Edited.